src/main/java/Duke.java

```diff
@@ -1,10 +1,378 @@
+     * @return returns validity of the event statement.
+     */
+    private static boolean isValidEventInput(String input) {
+        return Pattern.matches("event [a-z0-9\\s]+\\b /at .+", input.toLowerCase());
+    }
+
+    /**
+     * This function gets the parameters for the Task subclasses by slicing input.
+     *
+     * @param parameters Array of string of fixed size 2 to store parameters for Task subclasses.
+     * @param input      User's input into command line.
+     * @param taskType   String that is pre-identified (DEADLINE/EVENT/TODO)
+     */
+    private static void getParameters(String[] parameters, String input, String taskType) throws InvalidTaskException {
+        switch (taskType) {
+        case TODO:
+            String[] todoParts = input.split("(?i)todo ");
+            parameters[DESCRIPTION] = todoParts[1];
+            break;
+        case DEADLINE:
+            String[] initDeadlineParts = input.split("(?i)deadline ");
+            String[] deadlineParts = initDeadlineParts[1].split(" /by ");
+            if (deadlineParts.length != 2) {
+                throw new InvalidTaskException();
+            }
+            parameters[DESCRIPTION] = deadlineParts[0];
+            parameters[DATETIME] = deadlineParts[1];
+            break;
+        case EVENT:
+            String[] initEventParts = input.split("(?i)event ");
+            String[] eventParts = initEventParts[1].split(" /at ");
+            if (eventParts.length != 2) {
+                throw new InvalidTaskException();
+            }
+            parameters[DESCRIPTION] = eventParts[0];
+            parameters[DATETIME] = eventParts[1];
+            break;
+        }
+        if (Objects.equals(parameters[DESCRIPTION], "")) {
+            throw new InvalidTaskException();
+        }
+    }
+
+    /**
+     * This function prints the welcome banner.
+     */
+    private static void printWelcomeBanner() {
+
+        String sadge = ":( :( :( :( :( :( :( :( :( :( :( :(\n"
+                + "D: D: D: D: D: D: D: D: D: D: D: D:\n"
+                + ":( :( :( :( :( :( :( :( :( :( :( :(\n"
+                + "D: D: D: D: D: D: D: D: D: D: D: D:\n"
+                + ":( :( :( :( :( :( :( :( :( :( :( :(\n"
+                + "D: D: D: D: D: D: D: D: D: D: D: D:\n";
+
+        String text = "SADGE DUKE";
+        System.out.println("Hello from\n" + sadge + text);
+        System.out.println(LINES);
+        System.out.println(GREETING);
+        System.out.println(LINES);
+    }
+
+    /**
+     * This function adds saved tasks to list.
+     */
+    private static void readSavedTasks() {
+        File dataFile = new File(FILE_PATH);
+        try {
+            Scanner lineScanner = new Scanner(dataFile);
+            while (lineScanner.hasNext()) {
+                AddLineTask(lineScanner.nextLine());
+            }
+        } catch (FileNotFoundException e) {
+            createFile();
+        }
+
+    }
+
+    private static void AddLineTask(String line) {
```

Comment:
Your method name should be in the camel case (`addLineTask`) instead of the pascal case. Also, a more meaningful name should be used to make your code easily traceable.